docker/service_web/init.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+
+
+./run-wsgi-dev
+./run-io-worker
+./run-cpu-worker
```

Comment:
On the actual server these are run each as a separate sytemd process.  Thus the instructions to run them each in a separate terminal.  

This is all a bit misleading - the core of the app will work with just running the uwsgi server.  The io and cpu workers process incoming data from PGE's servers - a work around to use multiple cores with python.

I'll also need to look for the flask entry point.

TLDR most devs would rather run the flask development server and not worry about the celery workers.  I added these scripts so that I could simulate the live server - in fact, it basically IS the production setup, just run from terminals instead of systemd.